components/global/main.tf

```diff
@@ -19,4 +19,7 @@ module "landing_zone" {
   certificate_key_vault_name = var.certificate_key_vault_name
   oms_env                    = var.oms_env
   certificate_name_check     = var.certificate_name_check
+  key_vault_resource_group   = data.azurerm_resource_group.key_vault.name
+  log_analytics_workspace_id = data.azurerm_log_analytics_workspace.log.id
```

Comment:
use the module instead https://github.com/hmcts/terraform-module-log-analytics-workspace-id/